README.md

```diff
@@ -1,4 +1,4 @@
-# Cloud iNfrastructure Telco Task Force (CNTT)
+# Anuket specifications
```

Comment:
This is the Title for the whole document.
```suggestion
# Anuket project
```

Reply:
Yes, but this README is in for Anuket specifications and the whole Anuket project.

Reply:
It is also for conformance test suites and implementation cookbooks

Reply:
I agree that Anuket Specifications is not perfect, but I do not have any better aggregation of the RM, RA, RI and RC sub projects. 

Maybe in the title we can use Anuket project and explain that this repo contains only the listed sub-projects. What do you think @pgoyal01 ?

Reply:
@CsatariGergely Thanks -- that works

Reply:
Fixed in 9fbd62c

Reply:
@lylavoie @CsatariGergely but it is not an OpenStack Testing Cookbook. Why not change the sentence to "check the Anuket conformance of your existing OpenStack deployments? You would be primarily interested in the _Testing Cookbook_ (with link to the chapter) for OpenStack based Cloud Infrastructure and the test case descriptions and selection for more details."

The RC1 chapter Title can also be changed